src/server/src/modules/message.ts

```diff
@@ -121,11 +126,17 @@ export const getMessages = async (userId1: number, userId2: number) => {
             'receiver.biography',
             'receiver.email',
         ])
-        .orderBy('message.date', 'ASC')
-        .getMany();
+        // Order from the newest to the oldest.
+        .orderBy('message.date', 'DESC')
```

Comment:
Just curious why this couldn't be an ASC in order to avoid the .reverse function call in line 139 below.

Reply:
Because take(N) returns top n elements, so if I do ASC, it will get top N old messages.